x-pack/plugins/apm/public/components/app/mobile/transaction_overview/index.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiPanel, EuiSpacer } from '@elastic/eui';
+import { replace } from 'lodash';
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { useApmServiceContext } from '../../../../context/apm_service/use_apm_service_context';
+import { useApmParams } from '../../../../hooks/use_apm_params';
+import { useTimeRange } from '../../../../hooks/use_time_range';
+import { AggregatedTransactionsBadge } from '../../../shared/aggregated_transactions_badge';
+import { MobileTransactionCharts } from '../../../shared/charts/transaction_charts/mobile_transaction_charts';
+import { TransactionsTable } from '../../../shared/transactions_table';
+
+export function MobileTransactionOverview() {
+  const {
+    query: {
+      environment,
+      kuery,
+      rangeFrom,
+      rangeTo,
+      transactionType: transactionTypeFromUrl,
+    },
+  } = useApmParams('/mobile-services/{serviceName}/transactions');
+
+  const { start, end } = useTimeRange({ rangeFrom, rangeTo });
+
+  const { transactionType, serviceName, fallbackToTransactions } =
+    useApmServiceContext();
+
+  const history = useHistory();
+
+  // redirect to first transaction type
+  if (!transactionTypeFromUrl && transactionType) {
+    replace(history, { query: { transactionType } });
+  }
+
+  // TODO: improve urlParams typings.
+  // `serviceName` or `transactionType` will never be undefined here, and this check should not be needed
+  if (!serviceName) {
+    return null;
+  }
```

Comment:
Do we still need this?

Reply:
Not sure, I need to check.

Reply:
it seems we don't need it. I'll clean it up